### PR TITLE
Make lisk_bridge use the jq included in lisk core - Closes #232

### LINKED ISSUE #232: lisk_bridge.sh will fail if jq package is not installed.
On a fresh ubuntu machine:

- i configured lisk-build
- i run `scripts/lisk_bridge.sh`
- and got the following error: `scripts/lisk_bridge.sh: line 51: jq: command not found`
- the scripts just halts

We need to add this to the readme file, or to the toolchain script.

### PR DESCRIPTION
**Pull request by Nazgolze**
Closes #232

### CODE REVIEW

File: scripts/lisk_bridge.sh [hunk 1]
```diff
@@ -57,7 +60,7 @@ extractConfig() {
 # Queries the `/api/loader/status/sync` endpoint
 # and extracts the height for evaluation.
 blockMonitor() {
-  BLOCK_HEIGHT="$(curl -s http://localhost:"$PORT"/api/loader/status/sync | cut -d':' -f 5 | cut -d',' -f 1)"
+	BLOCK_HEIGHT="$(curl -s http://localhost:"$PORT"/api/loader/status/sync | cut -d':' -f 5 | cut -d',' -f 1)"
```
Comment (ghost) [inline scripts/lisk_bridge.sh:62]: Wrong identation?

File: scripts/lisk_bridge.sh [hunk 2]
```diff
@@ -90,15 +93,16 @@ passphraseMigration() {
 		exit 1
 	fi
 
-	jq ".forging.defaultKey += \"$master_password\"" "$LISK_CONFIG" > new_config.json
+	$JQ ".forging.defaultKey += \"$master_password\"" "$LISK_CONFIG" > new_config.json
+	mv new_config2.json new_config.json
```
Comment (ghost) [inline scripts/lisk_bridge.sh]: That does not look right. Wrong pasted one line too many?

File: scripts/lisk_bridge.sh [hunk 3]
```diff
@@ -48,7 +51,7 @@ extractConfig() {
 	export PORT
 	PORT="$(grep "port" "$LISK_CONFIG" | head -1 | cut -d':' -f 2 | cut -d',' -f 1 | tr -d '[:space:]')"
 
-	readarray secrets < <(jq -r '.forging.secret | .[]' "$LISK_CONFIG")
+	readarray secrets < <($JQ -r '.forging.secret | .[]' "$LISK_CONFIG")
```
Comment (ghost) [inline scripts/lisk_bridge.sh]: Should be double-quoted. Did you run shellcheck?

**Review by ghost — DISMISSED** [review]

File: scripts/lisk_bridge.sh (hunk 3 above)
Comment (Nazgolze) [reply scripts/lisk_bridge.sh]: yeah, it didn't yell at me -- I'll add them though

File: scripts/lisk_bridge.sh (hunk 1 above)
Comment (Nazgolze) [reply scripts/lisk_bridge.sh:62]: This fixes indentation to be in line with the rest of the file (I should've put that in the commit message)

File: scripts/lisk_bridge.sh (hunk 3 above)
Comment (ghost) [reply scripts/lisk_bridge.sh]: Something must be wrong with shellcheck on Jenkins thenm, it's reporting 1 warning and 8 info messages on my machine.

**Comment by ghost** [comment]
@Nazgolze Look at this: https://jenkins.lisk.io/job/lisk-build-test-private/view/change-requests/job/PR-236/5/console ([Jenkinsfile](https://jenkins.lisk.io/job/lisk-build-test-private/view/change-requests/job/PR-236/5/replay/diff))

**Review by karmacoma — APPROVED** [review]

**Pull request merged**